_toc.yml

```diff
@@ -597,9 +598,9 @@ entries:
     entries:
       - file: docs/products/mysql/get-started
         title: Get started
-      - file: docs/products/mysql/concepts                   
-        title: Concepts                           
-        entries:                                  
+      - file: docs/products/mysql/concepts
```

Comment:
You seem to have added some whitespace fixes to this PR. Could add the fixes in a separate pull request please, just adding the changes relevant to the custom plan article in this one?

Reply:
My editor trims trailing whitespaces by default on save. AFAIK showing it in diff as changeset could be disabled.

Reply:
Rebased to remove the additional changes